cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -100,13 +100,6 @@ spec:
       # END_PROMETHEUS_TO_SD
       nodeSelector:
         beta.kubernetes.io/fluentd-ds-ready: "true"
-      tolerations:
-      - key: "node.alpha.kubernetes.io/ismaster"
-        effect: "NoSchedule"
-      - operator: "Exists"
```

Comment:
This removes the tolerations now, which it shouldn't. Please rebase & make sure they are not removed.